backend/src/integration/groovy/pl/jcommerce/ewypowiedzenia/api/DismissalNoticeControllerIntSpec.groovy

```diff
@@ -9,8 +14,18 @@ import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.
 class DismissalNoticeControllerIntSpec extends BaseIntSpec {
 
     def "should create dismissal notice"() {
+        given:
+        def dismissalNoticeDto = new DismissalNoticeDto(firstName: "Adam", lastName: "Kowalski",
+                homeAddress: new AddressDto(street: "Armii Czerwonej 5/7", postalCode: "41-506", city: "Chorzów"),
+                companyName: "JanuszeBiznesu",
+                companyAddress: new AddressDto(street: "Jaruzelskiego 351", postalCode: "40-600", city: "Katowice"),
+                agreementDate: LocalDate.of(2017, 1, 2),
+                dismissalPlace: "Katowice", dismissalDate: LocalDate.now(), dismissalPeriod: 3)
+
         when:
-        def response = performRequest(post("/generate"))
+        def response = performRequest(post("/generate", dismissalNoticeDto)
+                .content(toJson(dismissalNoticeDto))
+                .contentType(MediaType.APPLICATION_JSON))
```

Comment:
Will be also good to check contents of returned dto

Reply:
This will change with the implementation of file download.